evap/staff/tests/test_views.py

```diff
@@ -1896,6 +1896,7 @@ def test_emails_are_sent(self):
         self.assertEqual(len(mail.outbox), 2)
 
 
+@override_settings(EVALUATION_END_OFFSET_HOURS=26)
```

Comment:
We only need this setting changed for the one test method, right? In this case, could you move the decorator there so it doesn't affect the whole class but only the one test method?